formfactory/actions.py

```diff
@@ -58,17 +59,37 @@ def send_email(form_instance, **kwargs):
     try:
         from_email = cleaned_data.pop(kwargs["from_email_field"])
     except KeyError:
-        raise KeyError("No to-field param provided.")
+        raise KeyError("No from_email_field action param provided.")
```

Comment:
I would use my own custom error class. Keys don't usually have spaces in them, and you're passing a full sentence to something that would be interpreted as being a key.